lib/data/idAndTimestamp.dart

```diff
@@ -0,0 +1,10 @@
+class IdListAndTimeStamp {
+  final int timestamp;
+  final List idList;
```

Comment:
idlist is not self explanatopry! please comment something like "list of ids of all files/chunks which are to be transferred"